src/openrct2/drawing/Text.cpp

```diff
@@ -123,12 +124,26 @@ static void DrawText(
     DrawText(dpi, coords, paint, buffer);
 }
 
+void DrawTextBasic(rct_drawpixelinfo* dpi, const ScreenCoordsXY& coords, rct_string_id format)
+{
+    Formatter ft;
```

Comment:
I think this needs initialisation. (Also applies to the other two functions)

```suggestion
    Formatter ft{};
```

Reply:
> Why is this duplication/overload in the functions necessary?

You cannot use the constructor in the header when it is forward declared. Some function calls didn't used the same number of parameters so the duplication is needed.